packages/ra-ui-materialui/src/input/ReferenceArrayInput.spec.tsx

```diff
@@ -187,32 +192,32 @@ describe('<ReferenceArrayInput />', () => {
                 .querySelector('input');
 
         await waitFor(() => {
-            expect(getCheckbox1().checked).toEqual(true);
-            expect(getCheckbox2().checked).toEqual(false);
+            expect(getCheckbox1()?.checked).toEqual(true);
+            expect(getCheckbox2()?.checked).toEqual(false);
```

Comment:
Just a boy scout edition to remove a few typescript errors. Shouldn't I?

Reply:
You did not only change the type, your also changed what the code does.

And since the assertions are `toEqual()` and not `toStrictEqual()` this could change the test assertions too.

You should ignore them or add `ts-ignore` comments if they really bother you IMO